### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by sandervanhooft**
Preparing release `v1.1.0`, with support for Laravel 6.

### CODE REVIEW

File: src/Order/Invoice.php [hunk 1]
```diff
@@ -470,7 +471,10 @@ public function pdf(array $data = [], string $view = self::DEFAULT_VIEW)
      */
     public function download(array $data = [], string $view = self::DEFAULT_VIEW)
     {
-        $filename = $this->date()->month.'_'.$this->date()->year.'.pdf';
+        $filename = implode('_', [
+                $this->id,
+                Str::snake(config('app.name', '')),
```
Comment (sandervanhooft) [inline src/Order/Invoice.php]: Need to use "old" string helper here: `snake_case`.

**Comment by ciungulete** [comment]
when do you release v1.1.0?

**Comment by sandervanhooft** [comment]
Today / tomorrow

File: src/Helpers/helpers.php [hunk 2]
```diff
@@ -94,3 +94,63 @@ function mollie_object_to_money(object $object)
...
+                return true;
+            }
+        }
+
+        return false;
+    }
+
+
+
+    if (! function_exists('snake_case')) {
```
Comment (RobertBoes) [inline src/Helpers/helpers.php]: Seems like you've put this inside the if-statement above. Shouldn't this be outside the `if (! function_exists('ends_with'))`?

**Comment by ciungulete** [comment]
Ups. Thanks

File: src/Helpers/helpers.php (hunk 2 above)
Comment (ciungulete) [reply src/Helpers/helpers.php]: Ups!!! Thanks

Comment (sandervanhooft) [reply src/Helpers/helpers.php]: @ciungulete are you able to fix this (and push to this branch)? Otherwise I'll pick it up.

**Comment by sandervanhooft** [comment]
#57

**Pull request merged**